### PR TITLE
Created a dict with the highest value of dicts, where key was updated…

### PR DESCRIPTION
**Pull request by Malahov-Alexandr**
… by '_' if the key has been met the second time with the highest value

### CODE REVIEW

File: Collections/Dictionary.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#
+# Commit script to git repository and provide link as home task result.
+
+import random
+import string
+
+new_dict = {}
+final_dict = {}
+# Creating a tuple with the letters for future dicts
+list_of_letters = string.ascii_lowercase
```
Comment (arastopchin1) [inline Collections/Dictionary.py:19]: You do not really need to create one more variable here. Why just don't use string.ascii_lowercase instead?

File: Collections/Dictionary.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# Creating an empty list with a random empty dicts
+list_of_dicts = [{} for _ in range(random.randint(2, 10))]
+# for each dict from list of dicts
+for current_dict in list_of_dicts:
+    # count of iteration for numbers of keys
+    for i in range(random.randint(1, len(list_of_letters))):
+        # creating pairs keys and values, a key is taken from list if letters and assigned with value from 0 to 100
+        current_dict[random.choice(list_of_letters)] = random.randint(0, 100)
+# iteration for each dict in the list of dicts
+for current_dict in list_of_dicts:
```
Comment (arastopchin1) [inline Collections/Dictionary.py:29]: Just 'dict' instead of 'current_dict' will be better.

File: Collections/Dictionary.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    for rest_key in rest_dict:
+        # if the key is already in the new_dict then compare theis value and if new value is higher key is updated
+        # with '_' and the highest value
+        if rest_key in new_dict:
+            # comparing existed in the new_dict value and new value of the rest keys
+            if new_dict.get(rest_key) < rest_dict.get(rest_key):
+                # if a new key's value is higher than existed, current key has name "name+'_'" and new value
+                new_dict[rest_key + '_'] = rest_dict.get(rest_key)
+                # deleting old key from the dict
+                new_dict.pop(rest_key)
```
Comment (arastopchin1) [inline Collections/Dictionary.py:48]: Your program doesn't work as expected. I would recommend to remove this loop, check task again and fill your 'new_dict' as required during previous loop. Also 'new_dict' is a bad naming.

**Review by arastopchin1 — CHANGES_REQUESTED** [review]

**Comment by Malahov-Alexandr** [comment]
fixed on another commite

**Pull request closed without merging**